src/ai/backend/manager/models/routing.py

```diff
@@ -34,6 +34,10 @@ class RouteStatus(Enum):
     PROVISIONING = "provisioning"
     FAILED_TO_START = "failed_to_start"
 
+    @classmethod
+    def active_route_statuses(cls) -> set["RouteStatus"]:
+        return {RouteStatus.HEALTHY, RouteStatus.UNHEALTHY, RouteStatus.PROVISIONING}
```

Comment:
Are you sure these are active statuses?
Also, it looks like "RouteStatus" can be used as a Self.

Reply:
It will depend on the situation, but since the meaning of `active` here implies that the depending session is alive and user has no intent of terminating the session I think it is legit to call such statuses as `active`.

Reply:
<img width="966" alt="image" src="https://github.com/user-attachments/assets/1eaa0baa-3ecd-4e1e-8674-10775d3d718f" />
Looks like mypy does not like the idea...